### PR TITLE
e2e: make sure the containers use iptables-nft

### PR DESCRIPTION
**Pull request by dougsland**
We should set as default iptables-nft, not legacy.

### CODE REVIEW

File: tests/e2e/lib/ContainerFile.template [hunk 1]
```diff
@@ -33,6 +33,7 @@ RUN dnf install -y \
 	gpgme-devel \
 	libseccomp-devel \
 	iptables-services \
+	iptables-nft \
```
Comment (Yarboa) [inline tests/e2e/lib/ContainerFile.template:36]: If you replace both
You can skip L68,

Tried it 
```
podman exec -it control bash -c "iptables --version"
iptables v1.8.8 (nf_tables)
```

Comment (dougsland) [reply tests/e2e/lib/ContainerFile.template:36]: @Yarboa what you meant replace both?

Comment (Yarboa) [reply tests/e2e/lib/ContainerFile.template:36]:
```suggestion
	iptables-nft \
	iptables-services \
```

Comment (Yarboa) [reply tests/e2e/lib/ContainerFile.template:36]: @dougsland ^^

File: tests/e2e/lib/ContainerFile.template [hunk 2]
```diff
@@ -64,6 +65,8 @@ RUN dnf install -y \
 	net-tools \
 	iproute
 
+RUN dnf remove -y iptables-legacy
```
Comment (Yarboa) [inline tests/e2e/lib/ContainerFile.template:68]:
```suggestion

```

Comment (Yarboa) [reply tests/e2e/lib/ContainerFile.template:68]: You can remove it

**Review by Yarboa — COMMENTED** [review]
Please add Signed by in your commit

**Pull request closed without merging**